zeq/event.h

```diff
@@ -59,7 +59,9 @@ class Event
     Event& operator=( const Event& ) = delete;
 
     friend class detail::Subscriber;
-    void setData( const void* data, const size_t size );
+
+    /** Set the a raw buffer as event data. */
```

Comment:
grammar: Set the a raw... 
Besides, it's a private function, so I would not put a documentation for it, unless you see value for it.